app/services/fees/one_off_service.rb

```diff
@@ -109,6 +109,7 @@ def apply_provider_taxes(fees_result)
 
         res = Fees::ApplyProviderTaxesService.call(fee:, fee_taxes:)
         res.raise_if_error!
+        fee.save
```

Comment:
Can you please explain which data are not stored on fee? If we need it, can you please use `save! `to run validations?

Reply:
not saved taxes_rate on fee was causing a bug. Will update to `save!`